src/Common/CosmosClient.ts

```diff
@@ -109,7 +109,8 @@ export function client(): Cosmos.CosmosClient {
     (options as any).plugins = [{ on: "request", plugin: requestPlugin }];
   }
 
-  if (userContext.features.enablePriorityBasedThrottling && userContext.apiType === "SQL") {
+  const enablePriorityBasedExecution: boolean = userContext.databaseAccount?.properties?.enablePriorityBasedExecution;
```

Comment:
Why create a variable here if it's only used on the next line?
I would recommend:
if ( userContext.databaseAccount?.properties?.enablePriorityBasedExecution
     && userContext.apiType === "SQL") {...

Reply:
Changes have been made.